pytorch_lightning/trainer/logging.py

```diff
@@ -75,6 +76,8 @@ def log_metrics(self, metrics, grad_norm_dic, step=None):
             self.logger.agg_and_log_metrics(scalar_metrics, step=step)
             self.logger.save()
 
+            # track the logged metrics
+            self.logged_metrics = scalar_metrics
```

Comment:
@williamFalcon naming metrics in ckpt filename is not possible if not using a logger?